astropy/conftest.py

```diff
@@ -27,12 +27,14 @@
     # This is a workaround for the OpenSSL deprecation warning that comes from
     # the `requests` module. It only appears when both asdf and sphinx are
     # installed. This can be removed once pyopenssl 1.7.20+ is released.
-    modules_to_ignore_on_import=['requests'],
+    # It also ignores collections.abc warning from html5lib that has been fixed
+    # in dev but not in release (html5lib/html5lib-python#419)
+    modules_to_ignore_on_import=['requests', 'html5lib'],
```

Comment:
Note to self: Change `html5lib` to `bleach`. Turns out `bleach` bundles `html5lib` code. Sigh.